.github/workflows/crawler.yaml

```
@@ -0,0 +1,42 @@
+      - main
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  algolia_crawl:
+    name: Algolia Recrawl
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)